features/networking/operator.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Operator related networking scenarios
+  
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-22704
+  @admin
+  Scenario: The clusteroperator should be able to reflect the network operator version corresponding to the OCP version
+    Given I have a project
+    #Getting OCP version
+    When I run the :get admin command with:
```

Comment:
you don't need to be admin to run these commands

Reply:
Also, we prefer to avoid doing steps like `I run the :get command with` but with a more OO approach in getting these properties.

Reply:
@anuragthehatter, you can shorten this sceanrio once #127 is merged.  For example, you can replace line 9-13 with the following: `The evaluation of `cluster_version('version').version` is stored in the :ocp_version clipboard`

Reply:
Right, Peter,  admin is not necessarily required. Regarding #127, yep that will be useful and i similar one would be also useful if designed for operator version as there are numerous operators we can check the version for. I can look at designing that step definition.

Reply:
Oh i noticed #126 would also support clusteroperator if merged. i will wait. Thanks!

Reply:
@anuragthehatter, my mistake, it seems that both `oc get clusterversion` and `oc get clusteroperator'  do need to be executed as admin user.

Reply:
Yep, i assumed that Peter. Thanks!